ui/pages/providers/[id].js

```diff
@@ -151,11 +279,7 @@ export default function ProvidersEditDetails() {
                 className='px-6 py-5 text-left first:pr-6 first:pl-0'
               >
                 <div className='text-2xs text-gray-400'>{g.label}</div>
-                <span
-                  className={`text-sm ${
-                    g.font ? g.font : 'font-medium'
-                  } text-gray-800`}
-                >
+                <span className='text-sm font-medium text-gray-800'>
```

Comment:
why did we remove the `g.font`?

Reply:
Thought I added this on accident, restored